pkg/mkfifo/mkfifo.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mkfifo
+
+import "syscall"
+
+type Mkfifo struct {
+	Paths []string
+	Mode  uint32
+}
+
+func (m *Mkfifo) Exec() error {
```

Comment:
This is simple enough that I don't think it warrants a package.

Reply:
I agree with Chris here, I think its simple enough that if you need it from somewhere else we could just reimplement it.

Reply:
or, OTOH, we could add FIFO as one of the things we support in pkg/u-root/util/root.go, along with directories, files, etc.